server/cherry_pick.go

```diff
@@ -41,7 +41,7 @@ func (s *Server) handleCherryPick(eventIssueComment IssueComment) {
 		return
 	}
 
-	cmdOut, err := s.doCherryPick(args[1], pr)
+	cmdOut, err := s.doCherryPick(args[1], nil, pr)
```

Comment:
Getting the code for this case do is actually quite tricky. And because it's a rare case, I opted to not do this for now.